src/System.Diagnostics.StackTrace/tests/Configurations.props

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      uapaot-Windows_NT;
```

Comment:
> By doing that, then the netstandard configuration will be built everywhere except on netfx, which has a specific configuration.

Oh! I understand now. That makes sense. My understanding was that you needed to specify each one, but netcoreapp somehow had some magic to connect it to netstandard if there is no netcoreapp configuration